paris-processor/src/main/java/com/airbnb/paris/processor/utils/ParisProcessorUtils.kt

```diff
@@ -1,45 +1,52 @@
 package com.airbnb.paris.processor.utils
 
-class ParisProcessorUtils {
+internal object ParisProcessorUtils {
```

Comment:
This is purposefully public. We access it in the Airbnb app to figure out the name of generated methods. Would you mind adding a comment saying it should be left this way?